catalogue/webapp/pages/works.js

```diff
@@ -339,8 +353,9 @@ const Works = ({ works, searchParams }: Props) => {
                         />
                       </div>
                       <TogglesContext.Consumer>
-                        {({ relevanceRating }) =>
-                          relevanceRating && (
+                        {({ relevanceRating, relevanceRatingOptIn }) =>
```

Comment:
What's the difference between these two? Should we not just have 1?

Reply:
One is for the actual ratings (relevanceRating), the other is because Jenn wanted the whole thing behind a toggle so she could take a look at it all first